src/azul/project/hca/transformers.py

```diff
@@ -14,18 +14,54 @@
 
 def _project_dict(bundle: api.Bundle) -> dict:
     project: api.Project
+    laboratories: List[str]
+    contributors: List[str]
+
     project, *additional_projects = bundle.projects.values()
     reject(additional_projects, "Azul can currently only handle a single project per bundle")
+
+    laboratories = set()
+    contributors = set()
+
+    for contributor in project.contributors:
+        if contributor.laboratory:
+            laboratories.add(contributor.laboratory)
+
+        if contributor.contact_name:
+            contributors.add(contributor.institution)
+
     return {
```

Comment:
According to the spec it looks like we should also have project description and a list of contact names

Reply:
According to the conversation with Michael yesterday, this will become a list of `ProjectContact` (HumanCellAtlas/metadata-api).